packages/amplify-category-function/src/provider-utils/awscloudformation/utils/layerParams.ts

```diff
@@ -60,7 +60,7 @@ export interface OrgsLayer {
   orgs: string[];
 }
 
-class LayerState implements LayerMetadata {
+export class LayerState implements LayerMetadata {
```

Comment:
I don't see this export being used anywhere else in this PR. Was this intentional?